requirements/test.txt

```diff
@@ -21,3 +21,4 @@ nose
 mock
 tox
 coverage
+tenacity
```

Comment:
Errr... oops. This requirements doesn't belong in `test.txt`, but in `base.txt`.